src/modules/agora/AgoraClient.ts

```diff
@@ -108,6 +108,32 @@ export class AgoraClient implements FullNodeAPI
                 });
         });
     }
+
+    /**
+     * Request the latest PreImageInfo for an enrollment key
+     *
+     * @param {enroll_key} Hash of the UTXO used by the validator
```

Comment:
`     * @param enroll_key{Hash} Hash of the UTXO used by the validator`

Reply:
Please add @returns document.

Reply:
Looking at the official documentation, it should be `@param {type} name`

Reply:
Fixed documentation